<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>B P Kailash</title>

  <meta name="author" content="B P Kailash">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/seal_icon.png">
</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>B P Kailash</name>
              </p>
              <p>I am a student reasearcher at <a href="https://ibse.iitm.ac.in/">IBSE - Initiative for Biological Systems Engineering</a>.
                In the of summer 2018, I was a research intern at <a href="http://cdrumlab.org/">CDrum lab</a> supported by NUS.
                In summer 2017, I was a funded clinical diagnostics intern at <a href="https://research.medgenome.com/">MedGenome</a>
                guided by Dr <a href="https://in.linkedin.com/in/vivekgopalan">Vivek Gopalan</a>.
              </p>
              <p>
                I graduated from <a href="biotech.iitm.ac.in">Bhupat and Jyoti Mehta School of Biosciences, Department of Biotechnology</a>,
                <a href="iitm.ac.in">Indian Institute of Technology Madras</a> with a Integrated B.Tech + M.Tech degree in Biological Engineering,
                specializing in Computational Biology. My master's thesis was advised by Professor <a href="https://maninarayanan.com/">Manikandan Narayanan</a> at the
                Bioinformatics and Integrative Data Science laboratory, Department of Computer Science and Engineering, IIT Madras.
              <p style="text-align:center">
                <a href="mailto:kailashbp10@gmail.com">Email</a> &nbsp/&nbsp
                <a href="data/KailashBP-CV.pdf">CV</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?user=Rj9Ef80AAAAJ&hl=en">Google Scholar</a> &nbsp/&nbsp
                <a href="https://www.linkedin.com/in/kailashbp10/">LinkedIn</a> &nbsp/&nbsp
                <a href="https://twitter.com/kailashbp10">Twitter</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/kailash.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/kailash.jpg" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">

              <heading>News</heading>
              <ul>
                <li> September 2020: Our paper on the iGEM 2018 Interlab study got published Nature Communications Biology! </li>
                <li> February 2020: Presented updated work of ChassiDex at EMBO symposium Designing Biological Systems. </li>
                <li> February 2020: Master's thesis work on Alzheimer's inter-tissue co-expression network dysregulation presented by advisor at CSHL conference - Development and 3D Modeling of the Human Brain! </li>
                <li> October 2019: Presented work on the spatial progression of Alzheimer's at BESCON. </li>
                <li> October 2019: Our paper on the iGEM 2018 Interlab study is now on bioRxiv! </li>
                <li> July 2019: Our paper on ChassiDex the host organism database is now on bioRxiv!
                <li> October 2018: Won the Gold Medal at the iGEM 2018 competition!
                <li> November 2017: Won the Silver Medal at the iGEM 2018 competition!
                <!-- <li> May 2018: I visited CDrum Lab, NUS as an undergraduate summer researcher.</li> -->
                <!-- <li> May 2017: I visited MedGenome as a undergraduate summer intern.</li> -->
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Research</heading>
              <p>
                I am a 23-year-old biotechnology enthusiast, excited about uncovering the secrets of the genome.
                Computational Biology and Synthetic Bioengineering are my fields of interest.
                I am particularly interested in analysing the human genome to understand the manifestation of complex diseases.
              </p>
           </td>
         </tr>
       </tbody></table>

       <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
           <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">
             <td style="padding:10px;width:10%;vertical-align:text-top">
               <div class="one">
                <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/ddp-collage.png'></div>
               </div>
             </td>
             <td style="padding:20px;width:75%;vertical-align:middle">
               <papertitle>Multi-tissue differential correalation analysis of Alzheimer's disease</papertitle>
               <br>
               <strong>Kailash B P</strong>,
               <a href="">Naga Venkata Sai Kumar</a>,
               <a href="">Srivatsan C R</a>,
               <a href="https://in.linkedin.com/in/philge-philip-phd-a598a414">Philge Philip</a>,
               <a href="https://scholar.google.com/citations?user=f5gTDzAAAAAJ&hl=en">Manikandan Narayanan</a>.
               <em>BESCON</em> 2019;
               <em>CSHL - Development and 3D Modeling of the Human Brain</em>, 2019.
               <br>
               <a href="https://github.com/philge/BIRDS/tree/master/Brain%20Disease%20networks%20version%202">Code hosted here</a>.
               <p>
                  The study aims to understand the inter-tissue network of gene-gene interactions at the transcriptional level,
                  we developed a framework to statistically infer the underlying correlation (co-expression) network structure among
                  genes in different brain tissues, and the rewiring of this gene co-expression network between healthy vs diseased
                  groups of individuals. We applied this framework to infer inter-tissue interactions in Alzheimerâ€™s Disease (AD)
                  across four different brain regions - Frontal pole, Superior temporal gyrus, Parahippocampal gyrus, and Inferior
                  frontal gyrus - by utilizing gene expression data available on these brain tissues/regions collected postmortem from
                  about 225 AD individuals with varying disease severity and 100 healthy controls. Known cell type-specific marker genes
                  are then used to estimate the frequency of different brain cell types in each individual and thereby assess the effect
                  of cellular composition on the identified inter-tissue gene networks.
               </p>

             </td>
           </tr>

       <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
           <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">
             <td style="padding:10px;width:10%;vertical-align:text-top">
               <div class="one">
                <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/interlab-study-design.png'></div>
               </div>
             </td>
             <td style="padding:20px;width:75%;vertical-align:middle">
               <papertitle>Robust Estimation of Bacterial Cell Count from Optical Density</papertitle>
               <br>
               <a href="https://scholar.google.com/citations?user=_ZSyxZ8AAAAJ&hl=en">Jacob Beal</a>,
               <a href="https://scholar.google.com/citations?user=koji-Q8AAAAJ&hl=en">Natalie G. Farny</a>,
               <a href="https://scholar.google.co.uk/citations?user=74oFmfIAAAAJ&hl=en&oi=sra">Traci Haddock-Angelli</a>,
               <a href="">Vinoo Selvarajah</a>,
               <a href="https://scholar.google.com/citations?user=VfgIt4MAAAAJ&hl=en">Geoff S. Baldwin</a>,
               <a href="">Russell Buckley-Taylor</a>,
               <a href="">Markus Gershater</a>,
               <a href="">Daisuke Kiga</a>
               <a href="https://scholar.google.com/citations?user=GVHMq88AAAAJ&hl=en">John Marken</a>
               <a href="">Vishal Sanchania</a>
               <a href="">Abigail Sison</a>
               <a href="https://scholar.google.com/citations?user=8sUv6OkAAAAJ&hl=en">Christopher T. Workman</a>,
               <strong>iGEM Interlab Study Contributor</strong>.
               <em>Comms Bio</em>, 2020.
               <br>
               <!-- <a href="https://github.com/google/night-sight/tree/master/docs">project page</a> -->
               <p>
                The interlab is a series of studies ( <a href="https://doi.org/10.1371/journal.pone.0150182">2014 and 2015 study</a>,  <a href="https://doi.org/10.1371/journal.pone.0199432">2016 study</a>,  <a href="https://doi.org/10.1038/s42003-020-01127-5">2018 study</a>) organized by the iGEM competition. It is one of the competition requirements to earn a bronze medal. The studies are aimed at addressing the problems frequenting reproducibility in science.
              </p>
              <p>
                The 2018 edition of which I was a part of, aimed to study the interlab variability in fluorescent measurements. A widely used and important technique to measure the optical density (OD) of bacterial growth is based on fluorescent measurements. So, it is essential to address the variability in these measurements to ensure reproducibility of experiments.
                The observations based on studying the data from 244 different labs is detailed in the paper.
            </p>
             </td>
           </tr>

           <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
               <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">
                 <td style="padding:10px;width:10%;vertical-align:middle">
                   <div class="one">
                    <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/chassidex-flashtalk.png'></div>
                   </div>
                 </td>
                 <td style="padding:20px;width:75%;vertical-align:middle">
                   <papertitle>ChassiDex: A microbial database useful for synthetic biology applications</papertitle>
                   <br>
                   <strong>Kailash B P</strong>,
                   <a href="">Karthik D</a>,
                   <a href="">Mousami Shinde</a>,
                   <a href="https://scholar.google.com/citations?user=IYzv-hgAAAAJ&hl=en">Nikhita Damaraju</a>,
                   <a href="http://orcid.org/0000-0003-1178-1327">Anantha Barathi Muthukrishnan</a>,
                   <a href="https://scholar.google.com/citations?user=VfgIt4MAAAAJ&hl=en">Shashi Bala Prasad</a>,
                   <a href="https://scholar.google.com/citations?user=s14r8k0AAAAJ&hl=en">Guhan Jayaraman</a>.
                   <em>bioRxiv</em>, 2019.
                   <br> The code for the project is available on <a href="https://github.com/ChassiDex">GitHub</a>.
                   <p>
                     ChassiDex is an open-source, easy to access database of host organisms and tools to work with them. Researchers are increasingly opting to work with unconventional organisms as they have unique advantages over common model organisms, but the characterization of these organisms haven't been documented extensively. ChassiDex provides information about growth and maintenance conditions of such different organisms with applications in synthetic biology, protocols, parts, vectors and other data required to work with them, all in one place.
                   </p>
                 </td>
               </tr>

       <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
           <tr>
           <td style="padding:20px;width:100%;vertical-align:middle">
             <heading>Teaching Experience</heading>
             <p>

             </p>
          </td>
        </tr>
      </tbody></table>

      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
          <td style="padding:20px;width:100%;vertical-align:middle">
            <heading>Other projects</heading>
            <p>

            </p>
         </td>
       </tr>
     </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:0px">
                <br>
                <p style="text-align:center;font-size:small;">
                  Design and source code from <a style="font-size:small;" href="https://jonbarron.info">Jon Barron's website</a>
                </p>
              </p>
            </td>
          </tr>
        </tbody></table>
      </td>
    </tr>
  </table>
</body>

</html>
