<!DOCTYPE HTML>
<html lang="en">
<head>

    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <title>B P Kailash</title>
    <meta name="author" content="B P Kailash">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="stylesheet_white.css">
    <link rel="icon" type="image/png" href="images/seal_icon.png">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" rel="stylesheet">
    <style>
        body {
      font-family: 'Roboto', sans-serif;
      line-height: 1.6;
      margin: 0;
      padding: 0;
      color: #333;
    }
    h2 {
      color: #007bff;
      font-size: 2.5em;
      text-align: center;
      margin-bottom: 20px;
    }
    h3 {
      font-size: 1.8em;
    }
    nav {
      width: 100%;
      background-color: #f8f9fa;
      padding: 10px 0;
      position: sticky;
      top: 0;
      z-index: 1000;
    }
    nav div {
      max-width: 800px;
      margin: auto;
      text-align: center;
    }
    nav a {
      margin: 0 15px;
      text-decoration: none;
      color: #007bff;
    }
    @media (max-width: 768px) {
      table, td {
        width: 100%;
        display: block;
      }
      img {
        max-width: 100%;
        height: auto;
      }
    }
    footer {
      background-color: #f8f9fa;
      padding: 20px;
      text-align: center;
    }
    a:hover {
      text-decoration: underline;
      color: #0056b3;
    }
    img:hover {
      transform: scale(1.05);
      transition: transform 0.3s ease;
    }
    .social-icons a {
      margin: 0 10px;
      font-size: 24px;
      color: #007bff;
      text-decoration: none;
      transition: color 0.3s;
    }
    .social-icons a:hover {
      color: #0056b3;
    }
    </style>
</head>

<body>
  <nav>
    <div>
      <a href="#about">About</a>
      <a href="#news">News</a>
      <a href="#research">Research</a>
      <a href="#awards">Awards</a>
      <a href="#teaching">Teaching</a>
      <a href="#community">Community</a>
      
    </div>
  </nav>

  <section id="about">
    <body>
        <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px"> 
            <td style="padding:0px">
              <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                <tr style="padding:0px">
                  <td style="padding:2.5%;width:63%;vertical-align:middle">
                    <p style="text-align:center">
                      <name>B P Kailash</name>
                    </p>
                    <p>I am currently a PhD candidate at the <a href="https://rajlab.org/">Raj Lab</a>, Icahn School of Medicine at Mount Sinai (ISMMS),
                      advised by Dr. <a href="https://scholar.google.com/citations?user=_1soEyEAAAAJ&hl=en">Towfique Raj</a>. My research focuses on
                      decoding the genetic regulation of gene expression and splicing in the human brain using computational approaches.
                    </p>
                    <p>
                      I graduated from the <a href="https://biotech.iitm.ac.in">Bhupat and Jyoti Mehta School of Biosciences, Department of Biotechnology</a>,
                      <a href="https://www.iitm.ac.in">Indian Institute of Technology Madras</a> with an Integrated B.Tech + M.Tech degree in Biological Engineering,
                      specializing in Computational Biology.
                      <div class="social-icons">
                        <a href="mailto:kailashbp10@gmail.com" title="Email"><i class="fas fa-envelope"></i></a>
                        <a href="https://drive.google.com/file/d/1coTu0adypxMUBmjIfUMQULzojr1cAlFG/view?usp=sharing" title="CV"><i class="fas fa-file-alt"></i></a>
                        <a href="https://scholar.google.com/citations?user=Rj9Ef80AAAAJ&hl=en" title="Google Scholar"><i class="fas fa-graduation-cap"></i></a>
                        <a href="https://www.linkedin.com/in/kailashbp10/" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
                        <a href="https://twitter.com/kailashbp10" title="Twitter"><i class="fab fa-twitter"></i></a>
                        <a href="https://youtube.com/c/KailashBP" title="YouTube"><i class="fab fa-youtube"></i></a>
                      </div>
                  </td>
                  <td style="padding:2.5%;width:40%;max-width:40%">
                    <a href="images/kailash-3.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/kailash-3.jpg" class="hoverZoomLink"></a>
                  </td>
                </tr>
              </tbody></table>
      
  </section>

  <section id="news">
    <h2>News</h2>
    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
        <td style="padding:20px;width:100%;vertical-align:middle">
          <ul>
            <li> November 2024: Gave a short-talk at Genome Informatics 2024, Wellcome Genome Campus. </li>
            <li> October 2023: Delivered a platform oral presentation at ASHG 2023 in Washington, DC. </li>
            <li> May 2024: Gave a short-talk at Jackson Laboratories Workshop on Long-Read Sequencing. </li>
            <li> October 2023: Published "Alzheimer’s disease rewires gene coexpression networks" in <i>npj</i> Systems Biology and Applications.</li>
            <li> July 2022: Defended my proposal and became a PhD Candidate. </li>
            <li> August 2021: Joined Icahn School of Medicine at Mount Sinai as a PhD Student. </li>
            <li> June 2021: Our meta-analysis paper on the iGEM Interlab studies is now published in Plos one. </li>
            <li> September 2020: Our paper on the iGEM 2018 InterLab study got published in Communications Biology! </li>
            <li> February 2020: Presented updated work of ChassiDex at EMBO symposium Designing Biological Systems. </li>
            <li> December 2019: Master's thesis work on Alzheimer's inter-tissue co-expression network dysregulation presented by advisor at CSHL conference - Development and 3D Modeling of the Human Brain! </li>
            <li> October 2019: Presented work on the spatial progression of Alzheimer's at BESCON. </li>
            <li> October 2019: Our paper on the iGEM 2018 InterLab study is now on bioRxiv! </li>
            <li> July 2019: Our paper on ChassiDex the host organism database is now on bioRxiv! </li>
            <li> October 2018: Won the Gold Medal at the iGEM 2018 competition! </li>
            <li> November 2017: Won the Silver Medal at the iGEM 2018 competition! </li>
            <li> May 2018: I visited CDrum Lab, NUS as an undergraduate summer researcher. </li>
            <li> May 2017: I visited MedGenome as a undergraduate summer intern. </li>
        </td>
      </tr>
    </tbody></table>
  </section>

  <section id="research">
    <h2>Research</h2>
    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
        <td style="padding:20px;width:100%;vertical-align:middle">
          <p>I am passionate about computational biology, focusing on analyzing the human genome to understand complex diseases.
            My work involves multi-ancestry meta-analysis, cell-type prioritization using snRNA-seq, and exploring the transcriptional complexity
            of the brain using long-read sequencing.
          </p>
       </td>
     </tr>
   </tbody></table>

   <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

       <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">

         <td style="padding:10px;width:10%;vertical-align:middle">
           <div class="one">
            <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/BigBrain.png'></div>
           </div>
         </td>
        
         <td style="padding:20px;width:75%;vertical-align:middle">
           <papertitle>BigBrain: Decoding the trans-regulatory architecture of expression and splicing using >11k postmortem human brain transcriptomes. </papertitle>
           <br>
           <strong>Kailash BP</strong>,
           <a href="https://scholar.google.com/citations?user=WczdUdsAAAAJ&hl=en">Aline Réal</a>,
           <a href="https://scholar.google.com/citations?user=8QiyLbEAAAAJ&hl=en">Winston H Cuddleston</a>,
           <a href="https://scholar.google.com/citations?user=wnPuwUgAAAAJ&hl=en">Benjamin Z Muller</a>,
           <a href="https://scholar.google.com/citations?user=FrmBNM4AAAAJ&hl=en">Beomjin Jang</a>,
           <a href="https://scholar.google.co.uk/citations?user=x6GuSDsAAAAJ&hl=en">Jack Humphrey</a>,
           <a href="https://scholar.google.co.uk/citations?user=N9QGV6YAAAAJ&hl=en">David A Knowles</a>,
           <a href="https://scholar.google.com/citations?user=_1soEyEAAAAJ&hl=en">Towfique Raj</a>.
           <em>ASHG</em> 2023;
           <em>JAX Long-read sequencing workshop</em> 2024;
           <em>Genome Informatics Wellcome Genome Campus</em> 2024.
           <br>
           <p>
            The transcriptional complexity of the human brain is genetically controlled through a network of <i>cis</i> and 
            <i>trans</i>-acting genetic regulatory elements. Numerous studies have elucidated the functional consequences of 
            <i>cis</i>-acting genetic variants and their ties to neurological disease risk. However, fewer studies have explored 
            <i>trans</i>-acting genetic variants, which often indirectly modulate gene expression and splicing through 
            <i>cis</i>-regulatory elements. Studying these indirect <i>trans</i>-regulated effects is particularly challenging due to their small 
             individual effect sizes and huge genome-wide multiple testing burden. So, to achieve the statistical power 
             required to map <i>trans</i>-acting variants influencing the transcriptome, we have uniformly processed a paired 
             transcriptomic-genetic resource called “BigBrain” containing 13,350 RNA-seq samples with corresponding genotype 
             data from 6,586 individuals from 12 publicly available cohorts. BigBrain includes data from healthy controls and 
             various brain disorders spanning several central nervous system regions and ancestries. Large-scale genome-wide 
             quantitative-trait-loci mapping of <i>trans</i>-acting genetic variants using BigBrain will enhance our understanding 
             of the genetic regulation of the human brain and nominate regulatory elements contributing to neurological 
             disease risk.
           </p>
         </td>

       </tr>

       <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

        <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">

          <td style="padding:10px;width:10%;vertical-align:text-top">
            <div class="one">
             <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/ddp-collage.png'></div>
            </div>
          </td>

          <td style="padding:20px;width:75%;vertical-align:middle">
            <papertitle>Multi-tissue differential correlation analysis of Alzheimer's disease</papertitle>
            <br>
            <strong>Kailash B P</strong>,
            <a href="https://www.cse.iitm.ac.in/profile.php?arg=MTg1Nw==">Naga Venkata Sai Kumar</a>,
            <a href="https://in.linkedin.com/in/srivatsan-c-r-a81a69122">Srivatsan C R</a>,
            <a href="https://in.linkedin.com/in/philge-philip-phd-a598a414">Philge Philip</a>,
            <a href="https://scholar.google.com/citations?user=f5gTDzAAAAAJ&hl=en">Manikandan Narayanan</a>.
            <em>BESCON</em> 2019;
            <em>CSHL - Development and 3D Modeling of the Human Brain</em> 2019.
             <!-- <a href="https://drive.google.com/file/d/18zzHav0D5_IB9pRTMcJniE7CXRbvAS0W/view"><font color="green">[POSTER1]</font></a> -->
             <!-- <a href="https://drive.google.com/file/d/1LYXDD5MU7kaucO843HZCt00DBLjN86dO/view"><font color="green">[POSTER2]</font></a> -->
            <br>
            <!-- <a href="https://github.com/philge/BIRDS/tree/master/Brain%20Disease%20networks%20version%202">Code hosted here</a>. -->
            <p>
               The study aims to understand the inter-tissue network of gene-gene interactions at the transcriptional level.
               We developed a framework to statistically infer the underlying correlation (co-expression) network structure among
               genes in different brain tissues, and the rewiring of this gene co-expression network between healthy vs diseased
               groups of individuals. We applied this framework to infer inter-tissue interactions in Alzheimer’s Disease (AD)
               across four different brain regions - Frontal pole, Superior temporal gyrus, Parahippocampal gyrus, and Inferior
               frontal gyrus - by utilizing gene expression data available on these brain tissues/regions collected postmortem from
               about 225 AD individuals with varying disease severity and 100 healthy controls. Known cell type-specific marker genes
               are then used to estimate the frequency of different brain cell types in each individual and thereby assess the effect
               of cellular composition on the identified inter-tissue gene networks.
            </p>
          </td>

        </tr>


       <tr onmouseout="nightsight_stop()" onmouseover="nightsight_start()">

         <td style="padding:10px;width:10%;vertical-align:text-top">
           <div class="one">
            <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/interlab-study-design.png'></div>
           </div>
         </td>
         <td style="padding:20px;width:75%;vertical-align:middle">
           <papertitle>Robust Estimation of Bacterial Cell Count from Optical Density</papertitle>
           <br>
           <a href="https://scholar.google.com/citations?user=_ZSyxZ8AAAAJ&hl=en">Jacob Beal</a>,
           <a href="https://scholar.google.com/citations?user=koji-Q8AAAAJ&hl=en">Natalie G. Farny</a>,
           <a href="https://scholar.google.co.uk/citations?user=74oFmfIAAAAJ&hl=en&oi=sra">Traci Haddock-Angelli</a>,
           <a href="">Vinoo Selvarajah</a>,
           <a href="https://scholar.google.com/citations?user=VfgIt4MAAAAJ&hl=en">Geoff S. Baldwin</a>,
           <a href="">Russell Buckley-Taylor</a>,
           <a href="">Markus Gershater</a>,
           <a href="">Daisuke Kiga</a>
           <a href="https://scholar.google.com/citations?user=GVHMq88AAAAJ&hl=en">John Marken</a>
           <a href="">Vishal Sanchania</a>
           <a href="">Abigail Sison</a>
           <a href="https://scholar.google.com/citations?user=8sUv6OkAAAAJ&hl=en">Christopher T. Workman</a>,
           <strong>iGEM InterLab Study Contributor</strong>.
           <em>Comms Bio</em>, 2020.
           <a href="https://www.nature.com/articles/s42003-020-01127-5"><font color="green">[PAPER]</font></a>
           <a href="http://2018.igem.org/Team:IIT-Madras/InterLab"><font color="green">[CONTRIBUTION]</font></a>
           <br>
           <!-- <a href="https://github.com/google/night-sight/tree/master/docs">project page</a> -->
           <p>
            The iGEM InterLab Study is a series of projects ( <a href="https://doi.org/10.1371/journal.pone.0150182">2014 and 2015 study</a>,
            <a href="https://doi.org/10.1371/journal.pone.0199432">2016 study</a>,  <a href="https://doi.org/10.1038/s42003-020-01127-5">2018 study</a>) organized
            by the iGEM competition, focused on addressing the problems frequenting reproducibility in science.
          </p>
          <p>
            The 2018 InterLab study of which I was a part of addresses the question, Can we reduce lab-to-lab variability in fluorescence measurements by normalizing to absolute cell count or colony-forming units (CFUs) instead of OD?
            A widely used and important technique to measure the optical density (OD) of bacterial growth spectrophotometry, is based on fluorescent measurements. So, it is essential to address the variability in these measurements to ensure reproducibility across labs.
            Recommendations to produce reliable and repeatable measurements based on studying the data from 244 different labs is detailed in the paper.
        </p>
        </td>
       </tr>

       <td style="padding:10px;width:10%;vertical-align:middle">
         <div class="one">
          <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/chassidex-flashtalk.png'></div>
         </div>
       </td>

       <td style="padding:20px;width:75%;vertical-align:middle">
         <papertitle>ChassiDex: A microbial database useful for synthetic biology applications</papertitle>
         <br>
         <strong>Kailash B P</strong>,
         <a href="">Karthik D</a>,
         <a href="">Mousami Shinde</a>,
         <a href="https://scholar.google.com/citations?user=IYzv-hgAAAAJ&hl=en">Nikhita Damaraju</a>,
         <a href="http://orcid.org/0000-0003-1178-1327">Anantha Barathi Muthukrishnan</a>,
         <a href="https://scholar.google.com/citations?user=VfgIt4MAAAAJ&hl=en">Shashi Bala Prasad</a>,
         <a href="https://scholar.google.com/citations?user=s14r8k0AAAAJ&hl=en">Guhan Jayaraman</a>.
         <em>bioRxiv, 2019; EMBO Symposium - Engineering meets evolution: Designing biological systems.</em>
            <a href="https://www.biorxiv.org/content/10.1101/703033v1"><font color="green">[PAPER]</font></a>
            <a href="https://github.com/ChassiDex"><font color="green">[CODE]</font></a>
            <a href="https://chassidex.org/"><font color="green">[WEBSITE]</font></a>
            <a href="https://drive.google.com/file/d/1GAmXcuMjUuVetO5mvV95lJzV1eUgrXvW/view"><font color="green">[POSTER]</font></a>
            <a href="https://drive.google.com/file/d/1SS0gF1Ksh5Le13BSfQt-d_iGdVPnpKjS/view?usp=sharing"><font color="green">[PRESENTATION]</font></a>
         <p>
           ChassiDex is an open-source, easy to access database of host organisms and tools to work with them. Researchers are increasingly opting to work with unconventional organisms as they have unique advantages over common model organisms, but the characterization of these organisms haven't been documented extensively. ChassiDex provides information about growth and maintenance conditions of such different organisms with applications in synthetic biology, protocols, parts, vectors and other data required to work with them, all in one place.
         </p>
       </td>
     </tr>

    </tbody>
    </table>

  <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

      <tr>

      <td style="padding:20px;width:75%;vertical-align:middle">
        <heading>Science Communication</heading>

        <p>
          Our work was featued in the following newspapers and newsletters -
          <a href="https://timesofindia.indiatimes.com/city/chennai/theyve-got-a-one-shot-solution-for-issues-know-what/articleshow/68204698.cms">Times of India</a>,
          <a href="https://www.newindianexpress.com/cities/chennai/2019/apr/17/iit-m-students-will-help-you-learn-biology-in-33-languages-1965229.html">New Indian Express</a>,
          <a href="http://sites.ieee.org/indiacouncil/files/2018/11/p67-p68.pdf">IEEE Newsletter</a>
          <a href="https://indiabioscience.org/columns/indian-scenario/the-journey-of-the-language-project">India Bioscience</a>
        </p>
   </td>
 </tr>
</tbody></table>

<table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
              <td style="padding:10px;width:10%;vertical-align:text-top">
                <div class="one">
                 <div class="two" id='nightsight_image'><img style="width:110%;max-width:150%" src='images/language-project-collage.png'></div>
                </div>
              </td>
              <td style="padding:20px;width:75%;vertical-align:middle">
                <papertitle>The Language Project</papertitle>
                <a href="https://indiabioscience.org/columns/indian-scenario/the-journey-of-the-language-project"><font color="green">[ARTICLE]</font></a>
                  <a href="http://bit.ly/lang_proj"><font color="green">[VIDEOS]</font></a>

                <br>
                <p>
                  Synthetic biology is a recent and unique field that raises   several ethical questions, and therefore, public engagement is essential.
                  India is a diverse country with 22 official languages where only 10.6% of the population understands English (2011 census).
                  The Language Project aims to break the language barrier to make science accessible to everyone.
                </p>
                <p>
                  Hence, we began with the goal to design an introductory course on the basics of synthetic biology in several Indian languages.
                  We have so far created more than 100 educational videos, across 26 languages (9 Indian vernacular languages) by collaborating with 13
                  other iGEM teams. The effort which began as part of the community outreach for the iGEM 2018 competition, continued for the iGEM 2019 competition.
                  The science communication inititative was recongized at the iGEM 2019 competition with a nomination for the Best Education and Public Engagement award!
                </p>

              </td>
            </tr>

     </td>
   </tr>
 </tbody></table>
  </section>

  <section id="awards">
    <h2>Awards</h2>
    <ul>
        <li> Received the bursary Award + Graduate Student Travel Award for Genome Informatics 2024 </li>
        <li>
          Selected by the American Society of Human Genetics (ASHG) Awards Committee as 
          <strong>one of 60</strong> predoctoral semifinalists among <strong>over 1,000 applicants</strong> for the 
          <strong><a href="https://drive.google.com/file/d/1hYBw_-z__qCx31ocdBWmrpht0yr48Aob/view?usp=sharing">2023 Trainee Awards for Excellence in Human Genetics Research</a></strong> 
          receiving a cash award of $750.
      </li>
        <li> Received scholarship + Travel award amounting to 2785$ to attend the Summer Institute of Statistical 
          Genetics 2023. </li>
        <li>
          <strong><a href="https://drive.google.com/file/d/1sAKGwR4dxGb-u0bglmoMngOZUCYqc822/view?usp=sharing">1 of 6 winners</a>/26</strong> selected teams at the <strong>Tata Consultancy Services (TCS) poster presentation competition</strong> at a national technical festival <strong>Shaastra 2019</strong>, for presenting work on the ChassiDex database.
      </li>
        <li> Led the IIT-Madras iGEM team to a Gold Medal at the <a href ="https://2018.igem.org/Competition/Results">iGEM 2018</a> competition. </li>
        <li> Won the silver medeal with the IIT-Madras iGEM team at the <a href ="https://2017.igem.org/Competition/Results">iGEM 2017</a> competition. </li>
        <li>
          <strong>1<sup>st</sup>/7</strong> teams at a bio-hackathon conducted by <strong>SVCE iGEM 2017 team</strong>, for developing a user-friendly web page to codon optimize protein sequences for different chassis, using Python library Flask, HTML, and CSS.
      </li>
      </ul>
  </section>

  <section id="teaching">
    <h2>Teaching</h2>
    <ul>
        <li>
            <strong>Teaching Assistant</strong> for <em>BSR-1012: Biomedical Science - Fall; (Nov - Dec 2024)</em>
        </li>
        <li>
            <strong>Teaching Assistant</strong> for <em>BIO6300: Introduction to R Programming; (Jul - Dec 2022)</em>
        </li>
        <li>
            <strong>Teaching Assistant</strong> for <em>BT3040: Bioinformatics; (Jan - Aug 2020)</em>
        </li>
        <li>
            <strong>Teaching Assistant</strong> for <em>CS6024: Algorithmic approaches to computational biology; (Jul - Dec 2019)</em>
        </li>
    </ul>
  </section>

  <section id="community">
    <h2>Community Involvement</h2>
    <ul>
        <li>
            <strong>PEERS</strong> <em>Self Development and Wellness Events, Mount Sinai (Oct 2022 – Present)</em>
        </li>
        <li>
            <strong>Center of Excellence in Youth Education (CEYE)</strong> <em>Diversity, Equity and Inclusion (DEI) initiative (Aug 2022)</em>
        </li>
        <li>
            <strong>Mount Sinai PhD application interviewee</strong> <em>(Feb 2022, Jan 2023, Feb 2024)</em>
        </li>
        <li>
            <strong>Trainee Health and Wellness (THAW)</strong> <em>Self Development and Wellness Events (Oct 2021 – Jun 2022)</em>
        </li>
        <li>
            <strong>National Service Scheme</strong> <em>Project representative, Blood Connect, IIT Madras (July 2019 – June 2020)</em>
    </ul>
  </section>

  <footer>
    <div class="social-icons">
      <a href="mailto:kailashbp10@gmail.com" title="Email"><i class="fas fa-envelope"></i></a>
      <a href="https://drive.google.com/file/d/1coTu0adypxMUBmjIfUMQULzojr1cAlFG/view?usp=sharing" title="CV"><i class="fas fa-file-alt"></i></a>
      <a href="https://scholar.google.com/citations?user=Rj9Ef80AAAAJ&hl=en" title="Google Scholar"><i class="fas fa-graduation-cap"></i></a>
      <a href="https://www.linkedin.com/in/kailashbp10/" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
      <a href="https://twitter.com/kailashbp10" title="Twitter"><i class="fab fa-twitter"></i></a>
      <a href="https://youtube.com/c/KailashBP" title="YouTube"><i class="fab fa-youtube"></i></a>
    </div>
    <p>Kailash BP's personal website</p>
  </footer>
</body>
</html>
